insights/tests/client/connection/test_checkin.py

```diff
@@ -54,6 +55,7 @@ def test_canonical_facts_request(get_proxies, post, get_canonical_facts, rm_conf
 @patch('insights.client.connection.InsightsUploadConf.get_rm_conf', return_value={})
 @patch(
     "insights.client.connection.InsightsConnection._init_session", Mock())
```

Comment:
This is another thing I did wrong when writing the original tests. 😢 

```suggestion
    "insights.client.connection.InsightsConnection._init_session")
```

would be the correct notation. It’s not related to your pull request. I just noticed it and it made me sad.